core/settings.py

```diff
@@ -93,6 +90,7 @@
     CAS_SERVER_URL = env("CAS_SERVER_URL")
     CAS_VERSION = env("CAS_VERSION")
     INSTALLED_APPS += ("django_cas_ng",)
```

Comment:
I hadn't noticed earlier that it is extending a list with a tuple.
```
>>> ["bb"] + ("cc",)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can only concatenate list (not "tuple") to list
```
The shortand works but it's awkward
```suggestion
    INSTALLED_APPS += ["django_cas_ng"]
```